travis/pubring.asc

```diff
@@ -0,0 +1,41 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
why is this file under the `travis` folder? We used the use Travis but now it's github actions.. maybe make a new separate directory named differently?